src/ControlStream.c

```diff
@@ -1598,11 +1598,16 @@ int startControlStream(void) {
     if (AppVersionQuad[0] >= 5) {
         ENetAddress remoteAddress, localAddress;
         ENetEvent event;
-        
+
         LC_ASSERT(ControlPortNumber != 0);
 
         enet_address_set_address(&localAddress, (struct sockaddr *)&LocalAddr, AddrLen);
+#ifdef __3DS__
+        // binding to wildcard port is broken on the 3DS, so we need to define a port manually
+        enet_address_set_port(&localAddress, 48002);
```

Comment:
Quick note here, I'm not sure if this is the best port to use by default? I chose it from the ports noted in the "Manual port forwarding" section of the Moonlight instructions, and it _seems_ to work, but I'm not sure if this is the default port we should normally expect for `localAddress`.

Reply:
Nevermind, I started using a global variable for defining the 3DS's binding port both here and in PlatformSockets.c